src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -888,6 +888,7 @@ def cleanse_crash(self,
 class AndroidLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
   """Android libFuzzer runner."""
   DEVICE_FUZZING_DIR = '/data/fuzz'
+  LIBFUZZER_TEMP_DIR = '/data/local/tmp'
```

Comment:
nit: maybe give it some libFuzzer specific name since we use it in libFuzzer launcher only?

Reply:
Thanks for clarifying!